AssemblyInfo.Shared.cs

```diff
@@ -24,7 +24,7 @@
 
 [assembly: AssemblyVersion("9.0.0")]
 [assembly: AssemblyFileVersion("9.0.0.0")]
```

Comment:
`AssemblyVersion` and `AssemblyFileVersion` stay the same, as they need to be in the format of major.minor.patch.revision.

`AssemblyInformationalVersion` is where we set the `-rc` suffix.

An example of this can be seen on [Microsoft.Extensions.Logging](https://nuget.info/packages/Microsoft.Extensions.Logging/9.0.0-preview.7.24405.7).

Reply:
All these changes will likely break [the release pipeline](https://dev.azure.com/sonarsource/DotNetTeam%20Project/_apps/hub/ms.vss-releaseManagement-web.cd-workflow-hub?definitionId=2&_a=environments-editor-preview). see e.g. here:
![image](https://github.com/user-attachments/assets/2caf75cb-8ca5-4636-9672-b20fc99a6c71)

To me, the whole "rc" effort is ruched without proper communication and planning.

Reply:
I am not sure if this will break during the release pipeline.
I don't disagree with you in general, but I am not sure how to proceed.

On the one hand, I would like to have a correct process for RC releases.
On the other hand, I prefer trying to do an RC now instead of a proper major release.

Do you have any suggestion?